mmv1/products/apigee/Organization.yaml

```diff
@@ -43,6 +43,8 @@ async:
 custom_code:
   encoder: 'templates/terraform/encoders/apigee_organization.go.tmpl'
   custom_import: 'templates/terraform/custom_import/apigee_organization.go.tmpl'
+sweeper:
+  identifier_field: organization
```

Comment:
I'm pretty sure this is incorrect since the apigee_organization resource doesn't have an "organization" field - maybe just leave this out for now?

```suggestion
```